c/meterpreter/source/common/common_command_ids.h

```diff
@@ -173,6 +173,7 @@
 #define COMMAND_ID_STDAPI_AUDIO_MIC_START 1115
 #define COMMAND_ID_STDAPI_AUDIO_MIC_STOP 1116
 #define COMMAND_ID_STDAPI_AUDIO_MIC_LIST 1117
+#define COMMAND_ID_STDAPI_SYS_CONFIG_UPDATE_TOKEN 1120
```

Comment:
This should keep the grouping and be moved up to line 117 with the other `COMMAND_ID_STDAPI_SYS_CONFIG*` definitions. It also looks like 1118 is the lowest unused value. How did you arrive at 1120? Are 1118-1119 in use some where I haven't seen?

Reply:
I moved the COMMAND_ID_STDAPI_SYS_CONFIG_UPDATE_TOKEN definition with the other COMMAND_ID_STDAPI_SYS_CONFIG* definitions. We found refs to 1118 and 1119 in MSF metasploit-framework\lib\rex\post\meterpreter\extensions\stdapi\command_ids.rb, that's why we used 1120

Reply:
Ah you're right, I see them here now. These last two aren't implemented in the Windows Meterpreter IIRC so that's why they're not defined here.

https://github.com/rapid7/metasploit-framework/blob/25e8cf8faf56059a6320e0e4df9c78e595f83293/lib/rex/post/meterpreter/extensions/stdapi/command_ids.rb#L130-L131

Thanks for clarifying!